test/e2e_env/multizone/inbound_communication/inbound_passthrough_disabled.go

```diff
@@ -0,0 +1,226 @@
+				testserver.WithNamespace(namespace),
+				testserver.WithMesh(mesh),
+				testserver.WithName("k8s-test-server-wildcard"),
+				testserver.WithEchoArgs("echo", "--instance", "k8s-bound-wildcard", "--ip", "0.0.0.0"),
+			)).
+			Install(testserver.Install(
+				testserver.WithNamespace(namespace),
+				testserver.WithMesh(mesh),
+				testserver.WithName("k8s-test-server-pod"),
+				testserver.WithEchoArgs("echo", "--instance", "k8s-bound-pod", "--ip", "$(POD_IP)"),
+				testserver.WithoutProbe(),
+			)).
+			Setup(env.KubeZone2),
+		).To(Succeed())
+	})
+	E2EAfterAll(func() {
+		Expect(env.KubeZone2.TriggerDeleteNamespace(namespace)).To(Succeed())
+		Expect(env.KubeZone2.DeleteMesh(mesh))
+		Expect(env.UniZone2.DeleteMeshApps(mesh)).To(Succeed())
+		Expect(env.Global.DeleteMeshApps(mesh)).To(Succeed())
+	})
+
+	It("should check communication k8s to k8s", func() {
+		// given
+		podName, err := PodNameOfApp(env.KubeZone2, "demo-client", namespace)
+		Expect(err).ToNot(HaveOccurred())
+
+		// when
+		response, err := client.CollectResponse(
+			env.KubeZone2, "demo-client", "k8s-test-server-wildcard.inbound-passthrough-disabled.svc.80.mesh",
+			client.FromKubernetesPod(namespace, "demo-client"),
+		)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.Instance).To(Equal("k8s-bound-wildcard"))
+
+		// when
+		response, err = client.CollectResponse(
+			env.KubeZone2, "demo-client", "k8s-test-server-localhost.inbound-passthrough-disabled.svc.80.mesh",
+			client.FromKubernetesPod(namespace, "demo-client"),
+		)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.Instance).To(Equal("k8s-bound-localhost"))
+
+		// when
+		_, _, err = env.KubeZone2.Exec(namespace, podName, "demo-client",
+			"curl", "-v", "-m", "3", "--fail", "k8s-test-server-pod.inbound-passthrough-disabled.svc.80.mesh")
+
+		// then
+		Expect(err).To(HaveOccurred())
+	})
+
+	It("should check communication universal to universal", func() {
+		// when
+		response, err := client.CollectResponse(
+			env.UniZone2, "uni-demo-client", "uni-test-server-wildcard.mesh",
+		)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.Instance).To(Equal("uni-bound-wildcard"))
+
+		// when
+		response, err = client.CollectResponse(
+			env.UniZone2, "uni-demo-client", "uni-test-server-wildcard-no-tp.mesh",
+		)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.Instance).To(Equal("uni-bound-wildcard-no-tp"))
+
+		// when
+		response, err = client.CollectResponse(
+			env.UniZone2, "uni-demo-client", "uni-test-server-localhost.mesh",
+		)
+
+		// then
+		Expect(err).ToNot(HaveOccurred())
+		Expect(response.Instance).To(Equal("uni-bound-localhost"))
+
+		// when
+		_, _, err = env.UniZone2.Exec("", "", "uni-demo-client",
+			"curl", "-v", "-m", "3", "--fail", "uni-test-server-containerip.mesh")
+
+		// then
+		Expect(err).To(HaveOccurred())
+	})
+
+	It("should check communication k8s to universal", func() {
```

Comment:
I'd consider splitting this to `Describe("k8s to universal communication")` and many `Its`

Same for other its and second file with e2e tests